projects/storefrontapp-e2e/src/page-objects/cmslib/header.po.ts

```diff
@@ -27,6 +27,15 @@ export class Header {
   readonly searchInput: ElementFinder = this.searchComponent.element(
     by.css('input.y-search-box__input')
   );
+  readonly languageSwitcher: ElementFinder = this.header.element(
+    by.css('#languageSelector')
+  );
+  readonly languageDe: ElementFinder = this.languageSwitcher.element(
+    by.css('option[value="de"]')
+  );
+  readonly languageEn: ElementFinder = this.languageSwitcher.element(
```

Comment:
Let's make it a function with lang parameter. In case we would test 5 languages, we would have to make 5 selectors. What do you think ?